minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -342,7 +342,7 @@ public void run() {
    *
    * @return the proxy
    */
-  public AlluxioProxyService getProxy() {
+  public ProxyProcess getProxy() {
```

Comment:
Rename these calls to avoid confusion since we will do things like `getWorker().getWorker`?